### DISCUSSION TITLE
Error when resuming from filament change

### DISCUSSION THREAD

**Discussion by OrByR** [Guilouz/Klipper-Flsun-Speeder-Pad#108] [General]
I ran in to the following error when trying to resume after loading a new roll after running out:
"Move exceeds maximum extrusion (250.112mm^2 vs 110.000mm^2) See the 'max_extrude_cross_section' config option for details"

I managed to save the print so far by uploading an edited gcode file - but would like to know how do I stop this from happening again?

**Comment by Winky1st** [comment]
Similar thing is going on here (v400) . the difference is that the error in my case is - after reloading filament she  starts and quit with "error print out of range".. I'm looking for the exact sentence but i  cant find it on logs.

**Comment by nialabert** [comment]
I had the same error when using the M600 macros.
It seems that when you press "resume" you should be patient and never press a second time "resume".


[Resume and Klipper](https://klipper.discourse.group/t/save-restore-gcode-state-in-updated-pause-resume-macros/1995)

A check, as sugested, a the begining of the "resume" macros should be a good idea.
